relay-general/src/store/mod.rs

```diff
@@ -63,6 +64,8 @@ pub struct StoreConfig {
 
     /// The SDK's sample rate as communicated via envelope headers.
     pub client_sample_rate: Option<f64>,
+
+    pub client_hints: ClientHints<String>,
```

Comment:
Added clienthints as a separate field due to backwards compatibility wrt serialization. If this is not necessary for StoreConfig, let me know